src/app/shared/session/session.query.ts

```diff
@@ -50,6 +50,9 @@ export class SessionQuery extends Query<SessionState> {
   gitHubAppInstallationLink$: Observable<string> = this.entryType$.pipe(
```

Comment:
Haven't checked, but is this still used if we always want to redirect to the landing page?

Reply:
I believe this is used for the "Refresh Organization" buttons in my workflows.